### PR TITLE
Feature/ab2d 505 filter data

### PR DESCRIPTION
**Pull request by lailasharshar**
### Summary

This is used to filter from a Explanation of Benefit object to one that is filtered to only provide the relevant data to the part D providers.

### CODE REVIEW

File: filter/src/main/java/gov/cms/ab2d/filter/EOBToAB2DEOB.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    /**
+     * Load the AB2dExplanationOfBenefit object from a JSON file
+     *
+     * @param path - the file name and path in the classpath containing the
+     *             Explanation of Benefit data retrieved from Blue
+     * @return the AB2D object
+     */
+    public static AB2DExplanationOfBenefit fromFile(String path) {
```
Comment (dkrylovsb) [inline filter/src/main/java/gov/cms/ab2d/filter/EOBToAB2DEOB.java]: The worker will have an EOB resource in-memory, so we need an extra method, such as `fromReader(Reader eobReader`).

File: filter/src/main/java/gov/cms/ab2d/filter/EOBToAB2DEOB.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        System.out.println("Loading file " + path);
```
Comment (dkrylovsb) [inline filter/src/main/java/gov/cms/ab2d/filter/EOBToAB2DEOB.java]: Let's use `slf4j` to make sure this message flows through the centralized logging.

File: filter/src/main/java/gov/cms/ab2d/filter/EOBToAB2DEOB.java (hunk 1 above)
Comment (dkrylovsb) [inline filter/src/main/java/gov/cms/ab2d/filter/EOBToAB2DEOB.java]: Let's rename this method to clearly communicate the fact that it really loads only from a _classpath_ resource, not an arbitrary file. `fromClasspath` or something along these lines.

File: filter/src/main/java/gov/cms/ab2d/filter/EOBToAB2DEOB.java (hunk 2 above)
Comment (lailasharshar) [reply filter/src/main/java/gov/cms/ab2d/filter/EOBToAB2DEOB.java]: This was actually a debug message that I forgot to delete while testing. Should I make it a log.debug message or just get rid of it?

Comment (dkrylovsb) [reply filter/src/main/java/gov/cms/ab2d/filter/EOBToAB2DEOB.java]: Getting rid of it works for me 👍

**Pull request merged**